src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -70,7 +70,7 @@ public async Task<double> CalculateAverageCarbonIntensityAsync(IDictionary props
         {
             var start = GetOffsetOrThrow(props, CarbonAwareConstants.Start);
             var end = GetOffsetOrThrow(props, CarbonAwareConstants.End);
-            var location = GetLocationOrThrow(props);
+            var location = GetSingleLocationOrThrow(props);
             ValidateDateInput(start, end);
```

Comment:
I know this is not part of the changes in this PR, but wondering if this log provides any value

Reply:
Honestly these debug messages aren't even useful IMHO, so I'm gonna axe them